.env.dev

```diff
@@ -2,6 +2,7 @@
 REACT_APP_FEATURE_OSMOSIS=true
 REACT_APP_FEATURE_PENDO=true
 REACT_APP_FEATURE_THOR=true
+REACT_APP_FEATURE_IDLE=false
```

Comment:
move to `.env.base`?

Reply:
Would feel safu-er having this in `.env.dev` only for now, since we're not going to toggle it on until quite some time